src/state/index.ts

```diff
@@ -1 +1,16 @@
-export type AccommoDateState = any;
+import { MatchProps } from "../components/matching/Match";
+
+export interface AccommoDateState {
+  matches: MatchProps[];
+}
+
+export const defaultState: AccommoDateState = {
+  matches: Array(5).fill([
+    {
+      chatStatusIcon: "string",
+      matchIsStarred: true,
+      matchName: "Tomas",
```

Comment:
The default state for every user shouldn't contain matches named "Tomas" 😉
Please use something like an empty array as the default state. 

This data could instead be moved to `test/components/matching/MatchingCockpit.spec.tsx`, where you can check whether everything is rendered the way you want it to be.